README.md

```diff
@@ -1,48 +1,70 @@
-# Welcome to Project Mercury
+# ![logo][]AI Shell (a.k.a Project Mercury)
 
-**Project Mercury** contains our latest CLI tool that provides an interactive shell session to chat
-with language models, creating an *AI Shell*. Users can use _agents_ to interact with different AI models, or other
-assistance providers, in a conversational manner. **Project Mercury** also provides a framework for
-creating AI agents.
+Welcome to the **AI Shell** repository! AI Shell is a CLI tool that brings the power of artificial
+intelligence directly to your command line! Designed to help you get command assistance from various
+AI assistants, AIShell is a versatile tool to help you become more productive in the command line.
+We call these various AI asssitant providers _agents_. You can use agents to interact with different
+generative AI models or other AI/ML/assistant providers in a conversational manner. This repo
+contains the code of the AI Shell engine, agents and details on how to create your own agent.
 
-Why the name **Project Mercury**? The name is inspired both by the Roman god of messages and the
-first human spaceflight by the US. This project is our first step into the new world of AI powered
-assistance and focuses on being the connection (or messenger) between the user and the AI model.
+Why the name **Project Mercury**? This was the original code name of the project and was inspired
+both by the Roman god of messages and the first human spaceflight by the US. This project is our
+first step into the new world of AI powered assistance and focuses on being the connection (or
+messenger) between the user and the AI model.
 
-This project is currently in the **alpha** phase. Expect many significant changes to the code as we
-experiment and refine the user experiences of this tool. We appreciate your feedback and patience as
-we continue our development.
+This project is currently in a very early **public preview** state. Expect many significant changes
+to the code as we experiment and refine the user experiences of this tool. We appreciate your
+feedback and patience as we continue our development.
 
 ![GIF showing demo of the AI Shell][04]
 
-## Installation
+## New to AI Shell?
+
+To learn more about AI Shell, we recommend you check out the overview page of the AI Shell
+documentation.
+
+There are two modes to use AI Shell, standalone and a side-by-side, integrated experience with
+PowerShell 7. For more information see,
+- [Get Started with AI Shell in PowerShell][15]
+- [Get Started with AI Shell (standalone)][16]
+
+## Getting AI Shell
+
+AI Shell is supported on Windows, MacOS and Linux, however the best experience you can have is with
+Windows, [PowerShell 7][11] and [Windows Terminal][14]. For more information see,
+[Installing AI Shell][13].
+
+## Locally Building AI Shell
 
 Some prerequisites for building an AI Shell:
 
-- Build script requires PowerShell v7.2 or newer versions
+- Build script requires PowerShell v7.4 or newer versions
```

Comment:
There is a tricky problem here:
the AIShell module depends on `System.Management.Automation` nuget package when building, so even though the module doesn't really ship anything from that nuget package, it's required to always use the latest version of that package for building. However, depending on the latest `S.M.A` package means it requires the latest PowerShell to use the module, for example, if we use the v7.4.6 version of `S.M.A`, it will require the PowerShell v7.4.6 to use the module.

I need to talk to Travis to see if we can get an exception to build against the v7.4.0 version of PowerShell, and if so the module can work with any 7.4 version PowerShell. But I don't put much hope on it. I think it's better to make `v7.4` a hyperlink to the latest 7.4.6 release page of PowerShell (https://github.com/PowerShell/PowerShell/releases/tag/v7.4.6).